firestore-bigquery-export/firestore-bigquery-change-tracker/src/bigquery/handleFailedTransactions.ts

```diff
@@ -0,0 +1,43 @@
+import * as firebase from "firebase-admin";
+import { FirestoreBigQueryEventHistoryTrackerConfig } from ".";
+
+if (!firebase.apps.length) {
+  firebase.initializeApp();
+  firebase.firestore().settings({ ignoreUndefinedProperties: true });
+}
+
+export default async (
+  rows: any[],
+  config: FirestoreBigQueryEventHistoryTrackerConfig,
+  e: Error
+): Promise<void> => {
+  const db = firebase.firestore();
+  const batchArray = [db.batch()];
+
+  let operationCounter = 0;
+  let batchIndex = 0;
+
+  rows.forEach((row) => {
```

Comment:
Nit: make this a `for of` loop to match the for of loop below that commits the batches